app/jobs/reports/irs_weekly_summary_report.rb

```diff
@@ -0,0 +1,55 @@
+require 'csv'
+
+module Reports
+  class IrsWeeklySummaryReport < BaseReport
```

Comment:
one more thing, I'm not seeing any changes to `config/job_configurations.rb` where the scheduling of the job happens